pkg/plugins/platform/utils.go

```diff
@@ -128,32 +128,6 @@ func ComposeFileExists(app App) bool {
 	return true
 }
 
-// Cleanup will clean up ddev apps even if the composer file has been deleted.
-func Cleanup(app App) error {
```

Comment:
Yeah, just has to handle syntax error in docker-compose.yml as well, which is what triggered this in the first place.